icaruscode/Generators/prodhepmc_icarus.fcl

```diff
@@ -0,0 +1,60 @@
+#include "simulationservices_icarus.fcl"
+#include "hepmcfilegen.fcl"
+#include "geometry_icarus.fcl"
+process_name: HepMCNuWro
+services:
+{
+  scheduler:    { defaultExceptions: false }    # Make all uncaught exceptions fatal.
+  # Load the service that manages root files for histograms.
+  TFileService:          {fileName: "MC_ldm_Aug27.root"}
+  TimeTracker:           {}
+  MemoryTracker:         {} # default is one
+  RandomNumberGenerator: {} #ART native random number generator
+#  FileCatalogMetadata:   @local::art_file_catalog_mc
+   @table::icarus_geometry_services
+}
+#Start each new event with an empty event.
+source:
+{
+  module_type: EmptyEvent
+  timestampPlugin: { plugin_type: "GeneratedEventTimestamp" }
+  maxEvents:   10000          # Number of events to create
+  firstRun:    1           # Run number to use for this file
+  firstEvent:  1           # number of first event in the file
+}
+# Define and configure some modules to do work on each event.
+# First modules are defined; they are scheduled later.
+# Modules are grouped by type.
+physics:
+{
+ producers:
+ {
+   generator: @local::standard_hepmcfilegen
+   rns:       { module_type: "RandomNumberSaver" }
+ }
+ #define the producer and filter modules for this path, order matters, 
+ #filters reject all following items.  see lines starting physics.producers below
+ simulate: [ rns, generator ]
+ #define the output stream, there could be more than one if using filters 
+ stream1:  [ out1 ]
+ #trigger_paths is a keyword and contains the paths that modify the art::event, 
+ #ie filters and producers
+ trigger_paths: [simulate] 
+ #end_paths is a keyword and contains the paths that do not modify the art::Event, 
+ #ie analyzers and output streams.  these all run simultaneously
+ end_paths:     [stream1]  
```

Comment:
These are actually redundant, and you may choose to omit them (matter of style).
```suggestion
```